plugins/github-issues/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@backstage/plugin-github-issues",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "prettier": "@spotify/prettier-config",
+  "dependencies": {
+    "@backstage/catalog-model": "^1.0.3",
+    "@backstage/core-components": "^0.10.1-next.0",
+    "@backstage/core-plugin-api": "^1.0.5-next.0",
+    "@backstage/integration": "^1.3.0-next.0",
+    "@backstage/plugin-catalog-react": "^1.1.3-next.0",
+    "@backstage/theme": "^0.2.15",
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.61",
+    "@octokit/rest": "^18.12.0",
```

Comment:
I think that we should be using the `octokit` library instead of the scoped ones. It's also going to reduce the amount of duplicate dependencies we bring in.

https://github.com/octokit/octokit.js

Reply:
The `scaffolder-backend` uses it if you need reference. It's pretty similar API.

Reply:
Updated here: [ffd5e47](https://github.com/backstage/backstage/pull/12875/commits/ffd5e47fb5c6338aa0e6965d687d0028e0baabb5)